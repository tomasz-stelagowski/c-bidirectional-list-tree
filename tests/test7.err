NODES: 2
NODES: 3
NODES: 4
NODES: 5
NODES: 6
NODES: 7
NODES: 8
NODES: 9
NODES: 10
NODES: 11
NODES: 12
NODES: 13
NODES: 14
NODES: 15
NODES: 16
NODES: 17
NODES: 18
NODES: 19
NODES: 20
NODES: 20
NODES: 20
NODES: 21
NODES: 20
NODES: 21
NODES: 19
NODES: 20
NODES: 20
NODES: 20
NODES: 21
NODES: 20
NODES: 21
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 21
NODES: 20
NODES: 21
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 21
NODES: 20
NODES: 21
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 21
NODES: 20
NODES: 21
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 21
NODES: 20
NODES: 21
NODES: 15
NODES: 16
NODES: 17
NODES: 18
NODES: 19
NODES: 20
NODES: 21
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 21
NODES: 20
NODES: 21
NODES: 7
NODES: 8
NODES: 9
NODES: 10
NODES: 11
NODES: 12
NODES: 13
NODES: 14
NODES: 15
NODES: 16
NODES: 17
NODES: 18
NODES: 19
NODES: 20
NODES: 19
NODES: 20
NODES: 20
NODES: 21
NODES: 20
NODES: 21
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 21
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 20
NODES: 20
