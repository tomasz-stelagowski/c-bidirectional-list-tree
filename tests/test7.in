ADD_NODE 0
SPLIT_NODE 0 1
ADD_NODE 2
SPLIT_NODE 0 1
ADD_NODE 3
SPLIT_NODE 0 1
ADD_NODE 3
SPLIT_NODE 0 6
ADD_NODE 4
SPLIT_NODE 6 4
ADD_NODE 10
SPLIT_NODE 0 1
ADD_NODE 10
SPLIT_NODE 0 12
ADD_NODE 1
SPLIT_NODE 10 13
ADD_NODE 16
SPLIT_NODE 0 12
ADD_NODE 4
RIGHTMOST_CHILD 8
RIGHTMOST_CHILD 12
ADD_NODE 5
DELETE_NODE 15
SPLIT_NODE 16 17
DELETE_SUBTREE 5
SPLIT_NODE 12 8
RIGHTMOST_CHILD 12
RIGHTMOST_CHILD 19
ADD_NODE 10
DELETE_NODE 4
SPLIT_NODE 2 3
DELETE_SUBTREE 8
RIGHTMOST_CHILD 14
RIGHTMOST_CHILD 12
RIGHTMOST_CHILD 9
ADD_NODE 1
DELETE_NODE 12
SPLIT_NODE 16 21
DELETE_SUBTREE 9
RIGHTMOST_CHILD 3
RIGHTMOST_CHILD 16
RIGHTMOST_CHILD 23
ADD_NODE 2
DELETE_NODE 14
SPLIT_NODE 10 16
DELETE_SUBTREE 26
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 28
RIGHTMOST_CHILD 16
ADD_NODE 21
DELETE_NODE 25
SPLIT_NODE 6 19
DELETE_SUBTREE 18
RIGHTMOST_CHILD 1
RIGHTMOST_CHILD 24
RIGHTMOST_CHILD 7
ADD_NODE 16
DELETE_NODE 31
SPLIT_NODE 2 27
DELETE_SUBTREE 2
ADD_NODE 10
SPLIT_NODE 0 22
ADD_NODE 1
SPLIT_NODE 10 13
ADD_NODE 36
SPLIT_NODE 30 10
DELETE_SUBTREE 29
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 34
RIGHTMOST_CHILD 36
ADD_NODE 21
DELETE_NODE 10
SPLIT_NODE 30 36
DELETE_SUBTREE 30
ADD_NODE 19
SPLIT_NODE 0 6
ADD_NODE 35
SPLIT_NODE 42 34
ADD_NODE 1
SPLIT_NODE 1 35
ADD_NODE 35
SPLIT_NODE 1 46
ADD_NODE 43
SPLIT_NODE 42 22
ADD_NODE 43
SPLIT_NODE 50 44
ADD_NODE 22
DELETE_SUBTREE 53
ADD_NODE 42
RIGHTMOST_CHILD 1
ADD_NODE 49
DELETE_NODE 35
SPLIT_NODE 1 48
DELETE_SUBTREE 34
RIGHTMOST_CHILD 1
RIGHTMOST_CHILD 42
RIGHTMOST_CHILD 6
ADD_NODE 22
DELETE_NODE 6
RIGHTMOST_CHILD 48
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 42
RIGHTMOST_CHILD 56
RIGHTMOST_CHILD 42
RIGHTMOST_CHILD 45
RIGHTMOST_CHILD 54
