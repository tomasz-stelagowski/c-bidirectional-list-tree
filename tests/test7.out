OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
-1
8
OK
OK
OK
OK
OK
22
-1
OK
OK
OK
OK
-1
22
-1
OK
OK
OK
OK
7
26
-1
OK
OK
OK
OK
18
23
21
OK
OK
OK
OK
-1
-1
-1
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
34
-1
37
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
48
OK
OK
OK
OK
56
54
19
OK
OK
-1
42
54
-1
54
-1
-1
